lib/cache/base.js

```diff
@@ -22,14 +22,30 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
+import { Counter } from 'prom-client';
+
 import { logger } from '../logger';
 import { getHash } from '../utils';
 
 const HashVersion = 'Compiler Explorer Cache Version 1';
 
+const get_counter = new Counter({
```

Comment:
argh my new projecst at work are all `like_this` and I keep naming things thus...will rename